content/blog/living-off-the-ipad-as-an-engineer/index.md

```diff
@@ -0,0 +1,110 @@
+
+I believe it’s common knowledge that you can’t just work out of a laptop form-factor all day without destroying your neck. Here the cheapest option would be to go with one of the pretty cool arm mount specifically designed to put the iPad right in front of your eyes or just a pile of books. This allows the expensive pixels to get their deserved amount of attention while also enabling instant video calling.
+
+While an external monitor is really comfortable, one thing to note is the lack of *full* external monitor support with iPadOS at the time of writing (version 14.4). Connecting a USB-C display like the Ultrafine essentially triggers an AirPlay mirroring of the iPad’s and is therefore really not as satisfying as a standard laptop+display setup, with the extended desktop for greater multitasking. Depending on its size and resolution, you’ll most likely end up with black bars around the mirrored video flux. Annoying even though you do get used to it. But they are ways around it.
+
+The workaround is indeed in the ability of iOS apps to specify how external monitor should be used. For instance, [Netflix](http://netflix.com) uses the iPad’s display for media controls while broadcasting the content onto the monitor. Luma Fusion has a mode for the video editor to stay on the iPad’s while live previewing on the monitor in full-screen.
+
+And in a clever workaround, a popular app called [Shiftscreen](http://shiftscreen.app) leverages these APIs to enable side-by-side web multitasking on the full external monitor, pretty convenient since nowadays a lot of work is actually happening within web apps like Google Docs or JIRA. On top of that, the app is still able to project the side-by-side browser on the external monitor even in Split Mode, therefore enabling another iOS app to be interacted with, such as a todo-list or direct messaging.
+
+![shiftscreen.app, the app’s webpage](shiftscreen.jpeg)
+
+I have to say, it’s an excellent option for specific tasks, but after a while, I just learned to love the mirrored interface, and I’m now rarely spending my time in apps that provide full monitor support. App Switching via Cmd+Tab or the gestures is hugely satisfying, plus I don’t think it has brought my productivity down, and on the flip side, it might have improved focus on the task at hand.
```

Comment:
I have to say, it’s an excellent option for specific tasks, but after a while I just learned to love the mirrored interface. Now I’m rarely spending my time in apps that provide full monitor support. App Switching via Cmd+Tab or the gestures is hugely satisfying, plus I don’t think it has brought my productivity down at all. In fact, it might have improved my focus on the task at hand.